docs/graphs/getting-started.md

```diff
@@ -72,9 +53,28 @@ FOR edge IN e
     INSERT edge IN flights
 ```
 
+## Create a Graphs
+
 Create a graph named airline with and edges in flights and both from and to vertices in cities.
 
-## Queries
+## Query your Graph
+
+Storing (and retrieving) a graph is one thing, but the actual problems only begin when we want to query information about a graph.
```

Comment:
Generally in technical documentation, we write for "you," because "we" is ambiguous and might be confusing.